src/main/dashboardServer/index.ts

```diff
@@ -0,0 +1,43 @@
+import path from 'path';
+
+import { app } from 'electron';
+import express from 'express';
+import { createProxyMiddleware } from 'http-proxy-middleware';
+
+import { proxyWsOpts, proxyOpts, proxyMetaOpts } from './proxyUtils';
+
+const host = '127.0.0.1';
+const port = 6120;
```

Comment:
We'll need to make all ports dynamic at some point… ideally we would stop hard-coding port numbers.

Reply:
Much agreed, I just don't think we'll be able to get there without some major changes to some dashboard implementation details.. I think that the changes we pushed through for this PR are going to help us get there in the future.